### ISSUE TITLE
Home >> Seção de cadastro na Newsletter

### ISSUE THREAD

**Issue by diogosp** [brasadesign/aboaterra-theme#6]
Eles já usam uma plataforma de newsletter específica. Que já tem integração com WP.

Assine nossa Newsletter - via plugin MDirector - http://www.mdirector.com.br/integracoes/wordpress/

<img width="862" alt="home_4_jpg" src="https://cloud.githubusercontent.com/assets/5107285/13110782/3c6a461a-d569-11e5-8b75-1e67780be016.png">

Link para a Home completa
https://drive.google.com/open?id=0B_zOFqExqtsKeURUQ09TMzFfVGs

**Comment by diogosp** [comment]
@matheusgimenez entrei em contato com o cliente pra falar sobre o erro no fechamento do HTML.
O cliente não quer mudar de plataforma. Ele indicou que falemos com um dos desenvolvedores do plugin para resolvermos isso e que eles corrijam lá no DEV deles.
Você poderia me indicar qual o problema que devo relatar pra eu encaminhar via e-mail?

**Comment by matheusgimenez** [comment]
Ficou um problema nisso:

Usei o shortcode pra exibir ele, porem, como ele utiliza IDs ao invés de classes no HTML, quando crio dois formulários (no caso da home, que tem no footer e no conteudo) ele envia somente o primeiro.

**Comment by LuwkasLima** [comment]
Alinhamos que eu vou explicar a limitacao ao Cliente...

@matheusgimenez nao conseguimos gerar mais de um shortcode com diferentes IDs?

**Comment by matheusgimenez** [comment]
Não. O id que falo é HTML, o atributo ID. Como o nome diz, ele deve ser unico, porem, os devs desse plugin não seguiram essa regra. Logo, se eu adiciono mais de 1 shortcode, vai ser gerado dois elementos HTML com o mesmo ID e na hora de enviar só funciona 1.

**Comment by LuwkasLima** [comment]
OK, vou validar com o cliente o caminho a seguirmos:

1 - Removemos ele de algum lugar da Home (mudança no Layout)
2 - Forkamos o Plugin e fazemos a correção (estimativa de horas necessaria)

**Comment by LuwkasLima** [comment]
@diogosp vou deixar esse ponto pra vc, porque nao julgo ser importante neste momento, deve ser incluido nas ultimas versoes...

**Comment by diogosp** [comment]
Removi via WIDGET a Newsletter d Footer da HOME. ele só ficará visível no Footer do restante do Site.
isto resolve a questão do campo de Newsletter no meio da Home

**Comment by diogosp** [comment]
@matheusgimenez Falta colocar o campo NOME. Você não colocou pq o plugin não habilita este campo ou foi um erro?

**Comment by diogosp** [comment]
Plugin não passa o campo NOME conforme aparece no layout. funcionalidade indisponível

**Comment by diogosp** [comment]
@matheusgimenez pergunta.
O cliente obteve uma resposta sobre o plugin. Que pra passar o campo NOME pra plataforma deles tem que ser via API. Mandou um manual (que confesso muito simples e tosco). Dá uma olhada pra ver se dá pra fazer isso mesmo e quantas horas SERIAM. Não é pra fazer ainda. Vou cobrar isso.

[API_INFORMAÇÃO GERAL_2016.pdf](https://github.com/brasadesign/aboaterra-theme/files/480911/API_INFORMACAO.GERAL_2016.pdf)

**Comment by diogosp** [comment]
@matheusgimenez definido que será o plugin do MAILCHIMP. Podemos ir em cima e finalizar.

https://wordpress.org/plugins/mailchimp/

**Comment by diogosp** [comment]
@matheusgimenez como ficou isso? preciso instalar o plugin pra funcionar? qual vc usou?

**Comment by matheusgimenez** [comment]
https://br.wordpress.org/plugins/mailchimp-for-wp/

**Comment by matheusgimenez** [comment]
Deixei configurado no dev, acredito ter esquecido de fazer isso antes:

http://dev.brasa.art.br/aboaterra/
